### DISCUSSION TITLE
Tache re-démarre

### DISCUSSION THREAD

**Discussion by Peluc64** [BOINC/boinc#5709] [Q&A]
**Describe the bug**
Les taches repartent en arriere de quelques secondes

**Steps To Reproduce**
1. Tout le temps
2. 

**Expected behavior**
Que les taches continuent à s'executer

**Screenshots**

**System Information**
 - OS: Mac Os 13,5
 - BOINC Version: 8,0,2

**Additional context**

**Comment by AenBleidd** [comment]
@Peluc64, thank you for your report.
Could you please give more details about the issue you have?
Thank you in advance

**Comment by Peluc64** [comment]
Bonjour, j’ai 4 taches actives. Par exemple, le temps écoulé pour la 1° est 25:00 ; Je regarde firefox en plein écran et quand je reviens sur Boinc, la tache est à 24:50. Elles sont actives tout le temps. Il est difficile de faire des copies d’écran vu que je ne sais pas exactement le moment où l’exécution des taches repart en arrière.
Cdlt.

**Comment by Peluc64** [comment]
Une autre information : après fermeture de Boinc et ré-ouverture, le temps écoulé des taches reste le même que lors de l’ouverture précédente !
Cdlt.

Cdlt.

**Comment by AenBleidd** [comment]
@Peluc64, pardon me, but I really don't understand what is the issue.
From the information you gave me, you want to say, that elapsed time of the tasks are changed very slow.
The reason for that might be in your settings.
If you go in your BOINC Manager -> Options -> Computing preferences, you will see the window similar to this:
![image](https://github.com/user-attachments/assets/94d6ff39-0853-435e-807e-03255efc7969)
Check it very carefully, because it may contain answer on your question.
For example, there is an option 'Suspend when computer is in use'. It means that if you actively doing something on your macbook, BOINC will suspend tasks execution.
